agri-store-react/src/App.js

```diff
@@ -7,6 +7,9 @@ import "./App.css";
 import { CartPage } from "./Pages/CartPage";
 import WishlistPage from "./Pages/WishlistPage";
 import Toast from "./Components/Toast/Toast";
+import AccountPage from "./Pages/AccountPage";
+import PageNotFound from "./Components/PageNotFound";
+import Authantication from "./Components/Authantication";

```

Comment:
You can create one index.js file inside pages, components folder and then export all files in index.js && then import all files from single file

Reply:
sure will do this in next PR